src/components/elements/default-tab.js

```diff
@@ -74,10 +75,10 @@ const TextWrapper = styled(Text)`
     font-weight: bold;
 
     @media ${device.tabletS} {
-        font-size: var(--text-size-sm);
+        font-size: ${({ font_size }) => font_size ?? 'var(--text-size-sm)'};
     }
     @media ${device.mobileM} {
-        font-size: var(--text-size-s);
+        font-size: ${({ font_size }) => font_size ?? 'var(--text-size-sm)'};
```

Comment:
`sm` for both tabletS and mobileM?

Reply:
It's suppose to be `s` not `sm` good catch 👍🏻